src/main/views/common/check-aml-details/check-aml-details.njk

```diff
@@ -0,0 +1,58 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% extends "layouts/default.njk" %}
+{% block main_content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+         <form class="form" action="" method="post">
+            {% if businessName %}
+                {% include "partials/business_name.njk" %}
+            {% endif %}
+            {% if firstName %}
+                {% include "partials/user_name.njk" %}
+            {% endif %}
+            <h1 class="govuk-fieldset__heading">
+               <label class="govuk-label govuk-label--l">{{ i18n.checkAmlDetailsTitle }}</label>
+            </h1>
+            <p class="govuk-body">{{ i18n.checkAmlDetailsText1 }}</p>
+            <p class="govuk-body">{{ i18n.checkAmlDetailsText2 }}</p>
+
+            {# Prepare table rows #}
+            {% set ROWS = [] %}
+            {% for body in amlSupervisoryBodies %}
+                {% set row = [
+                {text: body.amlSupervisoryBody},
+                {text: body.membershipId},
+                {html: "<a href=" + "/register-as-companies-house-authorised-agent/aml-membership-number#" + "membershipNumber_" + loop.index + ">change</a>"}
```

Comment:
Can we set the url to a variable and pass it in via the controller. Also "change" will need translated to welsh

Reply:
Addressed